packages/tealdeer/build.sh

```diff
@@ -1,12 +1,12 @@
-TERMUX_PKG_HOMEPAGE=https://dbrgn.github.io/tealdeer/
+TERMUX_PKG_HOMEPAGE=https://tealdeer-rs.github.io/tealdeer/
 TERMUX_PKG_DESCRIPTION="A very fast implementation of tldr in Rust"
 TERMUX_PKG_LICENSE="Apache-2.0, MIT"
 TERMUX_PKG_LICENSE_FILE="LICENSE-APACHE, LICENSE-MIT"
```

Comment:
This can also be removed since these are checked for during license installation.
```suggestion
```